Functions/Mock.ps1

```diff
@@ -743,14 +768,14 @@ function Invoke-MockInternal {
                         Write-PesterDebugMessage -Scope Mock "Original command is Set-Variable, patching the call."
                     }
                     if ($MockCallState['BeginBoundParameters'].Keys -notcontains "Scope") {
-                        $MockCallState['BeginBoundParameters'].Add( "Scope", 2)
+                        $MockCallState['BeginBoundParameters'].Add( "Scope", 3)
```

Comment:
This is due to the actual function definition being wrapped in an internal function (to leverage `$args` content)